spec/models/item_spec.rb

```diff
@@ -2,12 +2,16 @@
 
 RSpec.describe Item, type: :model do
 
+  let!(:item) {FactoryGirl.create(:item)}
+  let(:item2) {FactoryGirl.create(:item, price: 3,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.